client/my-sites/marketing/tools/index.tsx

```diff
@@ -112,6 +116,9 @@ export const MarketingTools: FunctionComponent = () => {
 		<Fragment>
 			{ ! purchases && <QueryUserPurchases userId={ userId } /> }
 			<PageViewTracker path="/marketing/tools/:site" title="Marketing > Tools" />
+			<Button onClick={ toggleWhatsNew }> Click Me!</Button>
+
+			{ showGuide && <WhatsNewGuide toggleWhatsNew={ toggleWhatsNew } /> }
```

Comment:
Something I don't think I understood before but am starting to understand now... the `WhatsNewGuide` component basically needs to be able to `toggleWhatsNew` to close the modal right? I wonder if it wouldn't make sense to just add an `onClose` prop to `WhatsNewGuide`, it might make more sense than `toggleWhatsNew` which sort of has weird semantics as far as data sharing goes and implies that the WhatsNewGuide might somehow toggle the guide open (which at least it's not currently able to do as it's not even rendered if `showGuide` is false).

Reply:
I was thinking about that @sarayourfriend! I went with the Toggle function since the other way would require (I think) two functions in the parent (openGuide closeGuide). Should I go with two functions anyways to get the cleaner semantics?

Reply:
FWIW I suggested the `toggle*` naming when discussing with @donlair, but feel free to go away from that and go with either a `close*` or `open*` function, or both if needed. The `toggle*` was just an example I gave in DM 😉

Reply:
Also, this approach does look so much simpler, love it!